pkg/arvo/app/spider.hoon

```diff
@@ -249,7 +249,8 @@
       ?+  wire  (on-arvo:def wire sign-arvo)
         [%thread @ *]  (handle-sign:sc i.t.wire t.t.wire sign-arvo)
         [%build @ ~]   (handle-build:sc i.t.wire sign-arvo)
-        [%bind ~]      `state
+        [%bind ~]              `state
+        [%conversion-cache *]  `state
```

Comment:
this looks unused

Reply:
The conversion-cache? Yes, we throw away the result, but we no-op so as to avoid a crash message in the default case of `?+`

Reply:
nevermind, I misread this